server.ts

```diff
@@ -39,10 +39,12 @@ app.prepare().then(() => {
         await handle(ctx.req, ctx.res);
     });
 
-    router.post("/token/:identity", async (ctx: any) => {
+    router.post("/token/:identity/:role", async (ctx: any) => {
         const identity = ctx.params.identity;
+        const role = ctx.params.role;
+        console.log(role);
         let permission = config.twilio.user;
-        if (identity === "business" || identity === "coach") {
+        if (role === "admin" || role === "employer") {
             permission = config.twilio.admin;
```

Comment:
Can you explain me what is `config.twilio.admin`?

Reply:
config.twilio.admin is the SID of the admin role, this condition give the admin rights only to admin and employer, later we'll can create a "coach role" and an "employer role" if we think there right have to be differents.

Reply:
can you add a comment somewhere to explain this